### DISCUSSION TITLE
two rendezvous servers

### DISCUSSION THREAD

**Discussion by hboetes** [rustdesk/rustdesk#8523] [Q&A] [answered]
Hi there,
we've set up our own rendezvous server. Works great, but we also have to work with external partners, and they don't have our rendezvous server configured. So then I have to stop rustdesk and move the configuration directory out of the way, and start rustdesk again. And once I'm done, I have to switch to our custom rendezvous server again.

Therefore, my kind request to implement multiple rendezvous servers.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/1258
